CMakeLists.txt

```diff
@@ -17,18 +17,27 @@ project(manifold LANGUAGES CXX)
 
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
+if(EMSCRIPTEN)
+  set(JSBIND_DEFAULT ON)
+else()
+  set(JSBIND_DEFAULT OFF)
+endif()
+
 set(MANIFOLD_PAR "NONE" CACHE STRING "Parallel backend, either \"TBB\" or \"NONE\"")
 set(MANIFOLD_FLAGS "" CACHE STRING "Manifold compiler flags")
+option(MANIFOLD_CROSS_SECTION "Build CrossSection for 2D support" ON)
 option(MANIFOLD_EXPORT "Build mesh export (via assimp) utility library" OFF)
-option(MANIFOLD_TEST "Enable testing suite" ON)
 # fuzztest is a rather large dependency
 option(MANIFOLD_FUZZ "Enable fuzzing tests" OFF)
 option(MANIFOLD_DEBUG "Enable debug tracing/timing" OFF)
-option(MANIFOLD_PYBIND "Build python bindings" OFF)
-option(MANIFOLD_CBIND "Build C (FFI) bindings" ON)
-option(MANIFOLD_JSBIND "Build js binding" ${EMSCRIPTEN})
 option(MANIFOLD_EXCEPTIONS "Build manifold with exception enabled" ON)
 option(BUILD_SHARED_LIBS "Build shared library" ON)
+include(CMakeDependentOption)
+cmake_dependent_option(MANIFOLD_TEST "Enable testing suite" ON "MANIFOLD_CROSS_SECTION" OFF)
```

Comment:
This is new cmake syntax to me. What does it mean? It looks like cross section defaults to off, but I assume I'm wrong?

Reply:
This means `MANIFOLD_TEST` is only enabled when `MANIFOLD_CROSS_SECTION` is `ON`. I think this is a bit too strong.

Reply:
Okay, I can see why that was chosen though. I wonder how many `#ifdef`s we'll have to add to get it to run? May be a good idea to rearrange the tests a bit to put anything requiring `CrossSection` near each other.